salt/runners/pkg.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+'''
+Package helper functions using ``salt.modules.pkg``
+
+.. versionadded:: FIXME
```

Comment:
This needs to be fixed. Unfortunately I haven't found docs to set this by myself.